01-shapefile-attributes.Rmd

```diff
@@ -235,15 +221,14 @@ lines_HARV$TYPE
 
 ```
 
+Notice that two of our attributes have space in them: stone wall and woods road.
```

Comment:
"Notice that two of our TYPE attribute values consist of two separate words: 
stone wall and woods road. There are really four TYPE values, not six TYPE
values. "

Does this clarify the issue?  When levels is printed out there is no way to differentiate between two values "woods" and "road" and a single value called "woods road" unless you are already familiar with the values.